<!DOCTYPE html>
<html lang="en">
  <!-- Head -->
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    
    <!-- Metadata, OpenGraph and Schema.org -->


<!-- Standard metadata -->
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<title>
  
  
    lizanalab's website
  
</title>
<meta name="author" content="Ludvig  Lizana">
<meta name="description" content="lizanalab's academic page for contact details, papers, and projects.
">










<!-- Bootstrap & MDB -->
<link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04">
<!-- <link rel="stylesheet" href="/assets/css/mdb.min.css?62a43d1430ddb46fc4886f9d0e3b49b8"> -->
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous">

<!-- Bootstrap Table -->
<link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css">

<!-- Fonts & Icons -->
<link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

<!-- Code Syntax Highlighting -->
<link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light">




<!-- Styles -->

  <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;">

<link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e">
<link rel="canonical" href="https://lizanalab/">

<!-- Dark Mode -->

  <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark">
  <script src="/assets/js/theme.js?7a860bb1099f1ce3f7e41588816ec311"></script>


<!-- GeoJSON support via Leaflet -->


  </head>

  <!-- Body -->
  <body class="fixed-top-nav ">
    <!-- Header -->
    <header>
  <!-- Nav Bar -->
  <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation">
    <div class="container">
      
      <!-- Navbar Toggle -->
      <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar top-bar"></span>
        <span class="icon-bar middle-bar"></span>
        <span class="icon-bar bottom-bar"></span>
      </button>

      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          

          <!-- About -->
          <li class="nav-item active">
            <a class="nav-link" href="/">about
              
                <span class="sr-only">(current)</span>
              
            </a>
          </li>

          <!-- Other pages -->
          
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
              
                <li class="nav-item ">
                  
                  <a class="nav-link" href="/publications/">publications
                    
                  </a>
                </li>
              
            
          
            
              
                <li class="nav-item ">
                  
                  <a class="nav-link" href="/pages/">epicon
                    
                  </a>
                </li>
              
            
          
            
              
                <li class="nav-item ">
                  
                  <a class="nav-link" href="/people/">people
                    
                  </a>
                </li>
              
            
          
            
          
            
          
            
              
                <li class="nav-item ">
                  
                  <a class="nav-link" href="/people_korean/">연구실 소개
                    
                  </a>
                </li>
              
            
          
            
          
            
          
          
            <!-- Toogle theme mode -->
            <li class="toggle-container">
              <button id="light-toggle" title="Change theme">
                <i class="fa-solid fa-moon"></i>
                <i class="fa-solid fa-sun"></i>
              </button>
            </li>
          
        </ul>
      </div>
    </div>
  </nav>
  
    <!-- Scrolling Progress Bar -->
    <progress id="progress" value="0">
      <div class="progress-container">
        <span class="progress-bar"></span>
      </div>
    </progress>
  
</header>


    <!-- Content -->
    <div class="container mt-5" role="main">
      
        <div class="post">
  <header class="post-header">
    <h1 class="post-title">
      
        lizanalab's website
      
    </h1>
    <p class="desc"></p>
  </header>

  <article>
    
      <div class="profile float-right">
        
          
          
          
          

<figure>
  <picture>
    <!-- Auto scaling with imagemagick -->
    <!--
      See https://www.debugbear.com/blog/responsive-images#w-descriptors-and-the-sizes-attribute and
      https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images for info on defining 'sizes' for responsive images
    -->
    
      <source class="responsive-img-srcset" srcset="/assets/img/group-action-480.webp 480w,/assets/img/group-action-800.webp 800w,/assets/img/group-action-1400.webp 1400w," sizes="(min-width: 800px) 231.0px, (min-width: 576px)
      30vw, 95vw" type="image/webp">
    
    <img src="/assets/img/group-action.jpg?710c8a271bb35770b2c830e5d27ed119" class="img-fluid z-depth-1
      rounded" width="100%" height="auto" alt="group-action.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">
  </source></picture>

  
</figure>

        
        
      </div>
    

    <div class="clearfix">
<p>The life sciences have witnessed an unprecedented surge in data, encompassing genomics, transcriptomics, proteomics, and imaging technologies. While statistical and machine-learning approaches have provided valuable insights, they often lack the ability to unveil cause-and-effect relationships. Therefore, our group takes a different approach. We specialize in developing mechanistic models that integrate diverse datasets that offers deeper insights into biological processes and more reliabile predictions.</p>

<p>In collaboration with experimentalists, we tackle a wide array of topics, including biological networks, gene regulation, epigenetics, aging, intermittent search problems, polymer physics, and DNA folding in the cell nucleus. Additionally, we engage in interdisciplinary collaborations with earth scientists, employing network models to calculate CO2 emissions in the Arctic.</p>

<p>Explore our website to learn more about our ongoing projects, publications, and opportunities for collaboration.</p>

</div>

    <!-- News -->
    
      <h2>
        <a href="/news/" style="color: inherit">news</a>
      </h2>
      <div class="news">
  
    
    <div class="table-responsive" style="max-height: 60vw">
      <table class="table table-sm table-borderless">
        
        
        
          <tr>
            <th scope="row" style="width: 20%">Oct 20, 2025</th>
            <td>
              
                <a class="news-title" href="/news/announcement_abisko/">The Arctic Meeting for Adaptive Mechanisms in Biological Systems</a>
              
            </td>
          </tr>
        
          <tr>
            <th scope="row" style="width: 20%">Oct 04, 2025</th>
            <td>
              
                New arXiv paper: <a href="https://arxiv.org/abs/2412.02524" rel="external nofollow noopener" target="_blank">General Resetting Theory for Group Avoidance</a> by Juhee Lee, Seong-Gyu Yang, <a href="https://hjpark.me" rel="external nofollow noopener" target="_blank">Hye-Jin Park</a>, and Ludvig Lizana.

              
            </td>
          </tr>
        
          <tr>
            <th scope="row" style="width: 20%">Oct 03, 2025</th>
            <td>
              
                Blog post <a href="https://www.umu.se/en/news/reunions-and-new-directions-for-korea-icelab-collaboration-_11984076/" rel="external nofollow noopener" target="_blank">Reunions and new directions for Korea-IceLab collaboration</a>.


              
            </td>
          </tr>
        
      </table>
    </div>
  
</div>

    



    <!-- Selected papers -->
    
      <h2>
        <a href="/publications/" style="color: inherit">selected publications</a>
      </h2>
      <div class="publications">
  <ol class="bibliography">
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="kim2022fractal" class="col-sm-8">
    <!-- Title -->
    <div class="title">Fractal and knot-free chromosomes facilitate nucleoplasmic transport</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Yeonghoon
            Kim
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Jae-Hyung
            Jeon
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Physical Review Letters</em>,  2022
    </div>
    <div class="periodical">
      <a href="https://arxiv.org/abs/2109.14933" rel="external nofollow noopener" target="_blank">arXiv</a>
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.128.038101" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Chromosomes in the nucleus assemble into hierarchies of 3D domains that, during interphase, share essential features with a knot-free condensed polymer known as the fractal globule (FG). The FG-like chromosome likely affects macromolecular transport, yet its characteristics remain poorly understood. Using computer simulations and scaling analysis, we show that the 3D folding and macromolecular size of the chromosomes determine their transport characteristics. Large-scale subdiffusion occurs at a critical particle size where the network of accessible volumes is critically connected. Condensed chromosomes have connectivity networks akin to simple Bernoulli bond percolation clusters, regardless of the polymer models. However, even if the network structures are similar, the tracer’s walk dimension varies. It turns out that the walk dimension depends on the network topology of the accessible volume and dynamic heterogeneity of the tracer’s hopping rate. We find that the FG structure has a smaller walk dimension than other random geometries, suggesting that the FG-like chromosome structure accelerates macromolecular diffusion and target-search.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="lundkvist2023forecasting" class="col-sm-8">
    <!-- Title -->
    <div class="title">Forecasting histone methylation by Polycomb complexes with minute-scale precision</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Moa J.
            Lundkvist
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Yuri B.
            Schwartz
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Science Advances</em>,  2023
    </div>
    <div class="periodical">
      <a href="https://www.biorxiv.org/content/10.1101/2023.06.16.545270v1.abstract" rel="external nofollow noopener" target="_blank">bioRxiv</a>
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://www.science.org/doi/abs/10.1126/sciadv.adj8198" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Animals use the Polycomb system to epigenetically repress developmental genes. The repression requires trimethylation of lysine 27 of histone H3 (H3K27me3) by Polycomb Repressive Complex 2 (PRC2), but the dynamics of this process is poorly understood. To bridge the gap, we developed a computational model that forecasts H3K27 methylation in Drosophila with high temporal resolution and spatial accuracy of contemporary experimental techniques. Using this model, we show that pools of methylated H3K27 in dividing cells are defined by the effective concentration of PRC2 and the replication frequency. We find that the allosteric stimulation by preexisting H3K27me3 makes PRC2 better in methylating developmental genes as opposed to indiscriminate methylation throughout the genome. Applied to Drosophila development, our model argues that, in this organism, the intergenerationally inherited H3K27me3 does not “survive” rapid cycles of embryonic chromatin replication and is unlikely to transmit the memory of epigenetic repression to the offspring. Our model is adaptable to other organisms, including mice and humans. Computational forecasts of histone methylation shed light on the intergenerational inheritance of epigenetic repression.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="hedstrom2024general" class="col-sm-8">
    <!-- Title -->
    <div class="title">Enhancer-Insulator Pairing Reveals Heterogeneous Dynamics in Long-Distance 3D Gene Regulation</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Lucas
            Hedström
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ralf
            Metzler
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>PRX life</em>,  2024
    </div>
    <div class="periodical">
      <a href="https://doi.org/10.48550/arXiv.2402.09209" rel="external nofollow noopener" target="_blank">arXiv</a>
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://journals.aps.org/prxlife/abstract/10.1103/PRXLife.2.033008" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Cells regulate fates and complex body plans using spatiotemporal signaling cascades that al- ter gene expression. Enhancers, short DNA sequences (50-150 base pairs), help coordinate these cascades by attracting regulatory proteins to enhance the transcription of distal genes by bind- ing to promoters. In humans, there are hundreds of thousands of enhancers dispersed across the genome, which poses a challenging coordination task to prevent unintended gene activation. To mit- igate this problem, the genome contains additional DNA elements, insulators, that block enhancer- promoter interactions. However, there is an open problem with how the insulation works, especially as enhancer-insulator pairs may be separated by millions of base pairs. Based on recent empirical data from Hi-C experiments, this paper proposes a new mechanism that challenges the common paradigm that rests on specific insulator-insulator interactions. Instead, this paper introduces a stochastic looping model where enhancers bind weakly to surrounding chromatin. After calibrating the model to experimental data, we use simulations to study the broad distribution of hitting times between an enhancer and a promter when there are blocking insulators. In some cases, there is a large difference between average and most probable hitting times, making it difficult to assign a typical time scale, hinting at highly defocused regulation times. We also map our computational model onto a resetting problem that allows us to derive several analytical results. Besides offering new insights into enhancer-insulator interactions, our paper advances the understanding of gene regulatory networks and causal connections between genome folding and gene activation.</p>
      </div>
    

    

    
  </div>
</div>
</li>
</ol>
</div>

    

    <!-- Social -->
    
      <div class="social">
        <div class="contact-icons">
  <a href="mailto:%6C%75%64%76%69%67.%6C%69%7A%61%6E%61@%75%6D%75.%73%65" title="email"><i class="fa-solid fa-envelope"></i></a>












  <a href="https://github.com/lizanalab" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-github"></i></a>


























  <a href="/feed.xml" title="RSS Feed"><i class="fa-solid fa-square-rss"></i></a>


</div>

        <div class="contact-note">Find us at <a href="https://icelab.se" rel="external nofollow noopener" target="_blank">Integrated Science Lab (IceLab)</a>, Umeå University, Umeå, Sweden.
</div>
      </div>
    
  </article>
</div>

      
    </div>

    <!-- Footer -->
    
  <footer class="fixed-bottom" role="contentinfo">
    <div class="container mt-0">
      © Copyright 2025
      Ludvig
      
      Lizana. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a>. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>.

      
      
    </div>
  </footer>



    <!-- JavaScripts -->
    <!-- jQuery -->
<script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
<script src="/assets/js/bootstrap.bundle.min.js"></script>
<!-- <script src="/assets/js/mdb.min.js"></script> -->
<script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    
  <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>


    

    

    

    

    

    

  <!-- Medium Zoom JS -->
  <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script>
  <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script>



<!-- Bootstrap Table -->
<script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script>

<!-- Load Common JS -->
<script src="/assets/js/no_defer.js?2930004b8d7fcd0a8e00fdcfc8fc9f24"></script>
<script defer src="/assets/js/common.js?4a129fbf39254905f505c7246e641eaf"></script>
<script defer src="/assets/js/copy_code.js?906085072389244b4db0c89c5b1dd28d" type="text/javascript"></script>

    
  <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>


  <script async src="https://badge.dimensions.ai/badge.js"></script>


    
  <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams',
      },
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>


    


    
  <!-- Scrolling Progress Bar -->
  <script type="text/javascript">
    /*
     * This JavaScript code has been adapted from the article
     * https://css-tricks.com/reading-position-indicator/ authored by Pankaj Parashar,
     * published on the website https://css-tricks.com on the 7th of May, 2014.
     * Couple of changes were made to the original code to make it compatible
     * with the `al-foio` theme.
     */
    const progressBar = $('#progress');
    /*
     * We set up the bar after all elements are done loading.
     * In some cases, if the images in the page are larger than the intended
     * size they'll have on the page, they'll be resized via CSS to accomodate
     * the desired size. This mistake, however, breaks the computations as the
     * scroll size is computed as soon as the elements finish loading.
     * To account for this, a minimal delay was introduced before computing the
     * values.
     */
    window.onload = function () {
      setTimeout(progressBarSetup, 50);
    };
    /*
     * We set up the bar according to the browser.
     * If the browser supports the progress element we use that.
     * Otherwise, we resize the bar thru CSS styling
     */
    function progressBarSetup() {
      if ('max' in document.createElement('progress')) {
        initializeProgressElement();
        $(document).on('scroll', function () {
          progressBar.attr({ value: getCurrentScrollPosition() });
        });
        $(window).on('resize', initializeProgressElement);
      } else {
        resizeProgressBar();
        $(document).on('scroll', resizeProgressBar);
        $(window).on('resize', resizeProgressBar);
      }
    }
    /*
     * The vertical scroll position is the same as the number of pixels that
     * are hidden from view above the scrollable area. Thus, a value > 0 is
     * how much the user has scrolled from the top
     */
    function getCurrentScrollPosition() {
      return $(window).scrollTop();
    }

    function initializeProgressElement() {
      let navbarHeight = $('#navbar').outerHeight(true);
      $('body').css({ 'padding-top': navbarHeight });
      $('progress-container').css({ 'padding-top': navbarHeight });
      progressBar.css({ top: navbarHeight });
      progressBar.attr({
        max: getDistanceToScroll(),
        value: getCurrentScrollPosition(),
      });
    }
    /*
     * The offset between the html document height and the browser viewport
     * height will be greater than zero if vertical scroll is possible.
     * This is the distance the user can scroll
     */
    function getDistanceToScroll() {
      return $(document).height() - $(window).height();
    }

    function resizeProgressBar() {
      progressBar.css({ width: getWidthPercentage() + '%' });
    }
    // The scroll ratio equals the percentage to resize the bar
    function getWidthPercentage() {
      return (getCurrentScrollPosition() / getDistanceToScroll()) * 100;
    }
  </script>


    

  </body>
</html>
