<!DOCTYPE html>
<html lang="en">
  <!-- Head -->
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    
    <!-- Metadata, OpenGraph and Schema.org -->


<!-- Standard metadata -->
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<title>
  
  
    
      publications | lizanalab's website
    
  
</title>
<meta name="author" content="Ludvig  Lizana">
<meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar.">










<!-- Bootstrap & MDB -->
<link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04">
<!-- <link rel="stylesheet" href="/assets/css/mdb.min.css?62a43d1430ddb46fc4886f9d0e3b49b8"> -->
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous">

<!-- Bootstrap Table -->
<link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css">

<!-- Fonts & Icons -->
<link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

<!-- Code Syntax Highlighting -->
<link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light">




<!-- Styles -->

  <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;">

<link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e">
<link rel="canonical" href="https://lizanalab/publications/">

<!-- Dark Mode -->

  <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark">
  <script src="/assets/js/theme.js?7a860bb1099f1ce3f7e41588816ec311"></script>


<!-- GeoJSON support via Leaflet -->


  </head>

  <!-- Body -->
  <body class="fixed-top-nav ">
    <!-- Header -->
    <header>
  <!-- Nav Bar -->
  <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation">
    <div class="container">
      
        <a class="navbar-brand title font-weight-lighter" href="/">
          lizanalab's website
        </a>
      
      <!-- Navbar Toggle -->
      <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar top-bar"></span>
        <span class="icon-bar middle-bar"></span>
        <span class="icon-bar bottom-bar"></span>
      </button>

      <div class="collapse navbar-collapse text-right" id="navbarNav">
        <ul class="navbar-nav ml-auto flex-nowrap">
          

          <!-- About -->
          <li class="nav-item ">
            <a class="nav-link" href="/">about
              
            </a>
          </li>

          <!-- Other pages -->
          
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
          
            
              
                <li class="nav-item active">
                  
                  <a class="nav-link" href="/publications/">publications
                    
                      <span class="sr-only">(current)</span>
                    
                  </a>
                </li>
              
            
          
            
              
                <li class="nav-item ">
                  
                  <a class="nav-link" href="/pages/">epicon
                    
                  </a>
                </li>
              
            
          
            
              
                <li class="nav-item ">
                  
                  <a class="nav-link" href="/people/">people
                    
                  </a>
                </li>
              
            
          
            
          
            
          
            
              
                <li class="nav-item ">
                  
                  <a class="nav-link" href="/people_korean/">연구실 소개
                    
                  </a>
                </li>
              
            
          
            
          
            
          
          
            <!-- Toogle theme mode -->
            <li class="toggle-container">
              <button id="light-toggle" title="Change theme">
                <i class="fa-solid fa-moon"></i>
                <i class="fa-solid fa-sun"></i>
              </button>
            </li>
          
        </ul>
      </div>
    </div>
  </nav>
  
    <!-- Scrolling Progress Bar -->
    <progress id="progress" value="0">
      <div class="progress-container">
        <span class="progress-bar"></span>
      </div>
    </progress>
  
</header>


    <!-- Content -->
    <div class="container mt-5" role="main">
      
        <div class="post">
  <header class="post-header">
    <h1 class="post-title">publications</h1>
    <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p>
  </header>

  <article>
    <!-- _pages/publications.md -->
<div class="publications">

<h2 class="bibliography">2025</h2>
<ol class="bibliography">
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="hedstrom2024exploring" class="col-sm-8">
    <!-- Title -->
    <div class="title">Exploring the benefits of DNA-target search with antenna</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Lucas
            Hedström
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Target Search Problems (Springer)</em>,  2025
    </div>
    <div class="periodical">
      <a href="https://doi.org/10.48550/arXiv.2311.11727" rel="external nofollow noopener" target="_blank">arXiv</a>
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://link.springer.com/chapter/10.1007/978-3-031-67802-8_24" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>The most common gene regulation mechanism is when a protein binds to a regulatory sequence to change RNA transcription. However, these sequences are short relative to the genome length, so finding them poses a challenging search problem. This paper presents two mathematical frameworks capturing different aspects of this problem. First, we study the interplay between diffusional flux through a target where the searching proteins get sequestered on DNA far from the target because of non-specific interactions. From this model, we derive a simple formula for the optimal protein-DNA unbinding rate, maximizing the particle flux. Second, we study how the flux flows through a target on a single antenna with variable length. Here, we identify a non-trivial logarithmic correction to the linear behavior relative to the target size proposed by Smoluchowski’s flux formula.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="hedstrom2025target" class="col-sm-8">
    <!-- Title -->
    <div class="title">Target search on networks-within-networks with applications to protein-DNA interactions</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Lucas
            Hedström
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Seong-Gyu
            Yang
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>New Journal of Physics</em>,  2025
    </div>
    <div class="periodical">
      <a href="https://arxiv.org/abs/2411.02660" rel="external nofollow noopener" target="_blank">arXiv</a>
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://iopscience.iop.org/article/10.1088/1367-2630/adf34e/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>We present a novel framework for understanding node target search in systems organized as hierarchical networks-within-networks. Our work generalizes traditional search models on complex networks, where the mean-first passage time is typically inversely proportional to the node degree. However, real-world search processes often span multiple network layers, such as moving from an external environment into a local network, and then navigating several internal states. This multilayered complexity appears in scenarios such as international travel networks, tracking email spammers, and the dynamics of protein-DNA interactions in cells. Our theory addresses these complex systems by modeling them as a three-layer multiplex network: an external source layer, an intermediate spatial layer, and an internal state layer. We derive general closed-form solutions for the steady-state flux through a target node, which serves as a proxy for inverse mean-first passage time. Our results reveal a universal relationship between search efficiency and network-specific parameters. This work extends the current understanding of multiplex networks by focusing on systems with hierarchically connected layers. Our findings have broad implications for fields ranging from epidemiology to cellular biology and provide a more comprehensive understanding of search dynamics in complex, multilayered environments.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="klaminder2025drying" class="col-sm-8">
    <!-- Title -->
    <div class="title">Drying by worming: deciphering how earthworms dry tundra soil</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Jonatan
            Klaminder
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Lucas
            Hedström
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Hanna
            Jonsson
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 2 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '2 more authors' ? 'Juhee Lee, Ludvig Lizana' : '2 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">2 more authors</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Geoderma</em>,  2025
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://www.sciencedirect.com/science/article/pii/S0016706125002599" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Deep-burrowing earthworms (anecic and endogeic species) can dry soils by reworking mineral soil layers. Although this ‘drying effect’ has been reported across many ecosystems, including the Fennoscandian tundra, little is known about the driving processes. In this study, we measure plant transpiration in combination with controlled experiments of water holding capacity and evaporation to assess drivers of soil–water losses in tundra soil as the result of endogeic and anecic earthworms. Our experimental system was a common garden experiment with shrub-dominated (heath) and forb-dominated (meadow) vegetation (N = 48), where long-term monitoring revealed drier soils due to the addition of earthworms. Although we found that tundra plant transpiration was highest during the peak growing season and that meadow soil had a higher field capacity, our earthworm treatment did not strongly affect these two parameters. Evaporation, on the other hand, was on average 14 % higher in the meadow with earthworms although no such effect was observed in the heath soil. Using a network model of macropore vapor transfer that measures evaporation effects, we found an increase in macropore conductance between the subsoil and the atmosphere and that the vaporization rate in relation to the diffusion rate controls the strength of the evaporation effect. Our findings underscore the need to account for evaporation due to the reworking of pore architectures by soil biota when predicting changes in soil–water availability.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="hedstrom2025identifying" class="col-sm-8">
    <!-- Title -->
    <div class="title">Identifying stable communities in Hi-C using multifractal network modularity</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Lucas
            Hedström
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Antón
            Carcedo
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>PLOS Complex Systems</em>,  2025
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://journals.plos.org/complexsystems/article?id=10.1371/journal.pcsy.0000053" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Chromosome capture techniques like Hi-C have expanded our understanding of mammalian genome 3D architecture and how it influences gene activity. To analyze Hi-C data sets, researchers increasingly treat them as DNA-contact networks and use standard community detection techniques to identify mesoscale 3D communities. However, there are considerable challenges in finding significant communities because the Hi-C networks have cross-scale interactions and are almost fully connected. This paper presents a pipeline to distill 3D communities that remain intact under experimental noise. To this end, we bootstrap an ensemble of Hi-C datasets representing noisy data and extract 3D communities that we compare with the unperturbed dataset. Notably, we extract the communities by maximizing local modularity (using the Generalized Louvain method), which considers the multifractal spectrum recently discovered in Hi-C maps. Our pipeline finds that stable communities (under noise) typically have above-average internal contac,t frequencies and tend to be enriched in active chromatin marks. We also find they fold into more nested cross-scale hierarchies than less stable ones. Apart from presenting how to systematically extract robust communities in Hi-C data, our paper offers new ways to generate null models that take advantage of the network’s multifractal properties. We anticipate this has a broad applicability to several network applications.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="solowiej2025competition" class="col-sm-8">
    <!-- Title -->
    <div class="title">Competition and cooperation: The plasticity of bacterial interactions across environments</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Josephine
            Solowiej-Wedderburn
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Jennifer T
            Pentz
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 3 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '3 more authors' ? 'Bjoern O Schroeder, Peter A Lind, Eric Libby' : '3 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">3 more authors</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>PLOS Computational Biology</em>,  2025
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1013213" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Bacteria live in diverse communities, forming complex networks of interacting species. A central question in bacterial ecology is whether species engage in cooperative or competitive interactions. But this question often neglects the role of the environment. Here, we use genome-scale metabolic networks from two different open-access collections (AGORA and CarveMe) to assess pairwise interactions of different microbes in varying environmental conditions (provision of different environmental compounds). By computationally simulating thousands of environments for 10,000 pairs of bacteria from each collection, we found that most pairs were able to both compete and cooperate depending on the availability of environmental resources. This modeling approach allowed us to determine commonalities between environments that could facilitate the potential for cooperation or competition between a pair of species. Namely, cooperative interactions, especially obligate, were most common in less diverse environments. Further, as compounds were removed from the environment, we found interactions tended to degrade towards obligacy. However, we also found that on average at least one compound could be removed from an environment to switch the interaction from competition to facultative cooperation or vice versa. Together our approach indicates a high degree of plasticity in microbial interactions in response to the availability of environmental resources.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="lizana2025chromatin" class="col-sm-8">
    <!-- Title -->
    <div class="title">Chromatin folding by the Polycomb group proteins and its elusive role in epigenetic repression</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Yuri B
            Schwartz
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>The FEBS Journal</em>,  2025
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://febs.onlinelibrary.wiley.com/doi/full/10.1111/febs.70199" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>The Polycomb system epigenetically represses selected developmental genes to enforce gene expression programs of differentiated cells. The system requires the coordinated action of dozens of structurally unrelated proteins assembled in two evolutionarily conserved polycomb repressive complexes, PRC1 and PRC2. Genes repressed by the Polycomb system are enriched in histone H3 trimethylated at lysine 27 (H3K27me3), an epigenetic mark that propagates the repressed state after DNA replication. Despite the impressive progress in dissecting molecular functions of the Polycomb group proteins, the fundamental questions of how the Polycomb system represses transcription or how the H3K27me3 mark is translated to benefit the repression are still open. Multiple observations indicate that the binding of PRC1, PRC2, and elevated H3K27me3 correlate with changes in the chromatin structure of target genes, which may be integral for the associated epigenetic repression. In this Review, we summarize our current understanding of these observations. We discuss the chromatin folding inside the loci repressed by the Polycomb system, consider molecular processes causing it and reflect upon its possible impact on transcription and epigenetic memory of the repressed state.</p>
      </div>
    

    

    
  </div>
</div>
</li>
</ol>
<h2 class="bibliography">2024</h2>
<ol class="bibliography">
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="lizana2024topology" class="col-sm-8">
    <!-- Title -->
    <div class="title">The scales, mechanisms, and dynamics of the genome architecture</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Yuri B.
            Schwartz
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Science Advances</em>,  2024
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://www.science.org/doi/full/10.1126/sciadv.adm8167" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Even when split into several chromosomes, DNA molecules that make up our genome are too long to fit into the cell nuclei unless massively folded. Such folding must accommodate the need for timely access to selected parts of the genome by transcription factors, RNA polymerases, and DNA replication machinery. Here we review our current understanding of the genome folding inside the interphase nuclei. We consider the resulting genome architecture at three scales and summarize the insights gained from recent experimental observations and diverse computational models.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="hedstrom2024general" class="col-sm-8">
    <!-- Title -->
    <div class="title">Enhancer-Insulator Pairing Reveals Heterogeneous Dynamics in Long-Distance 3D Gene Regulation</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Lucas
            Hedström
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ralf
            Metzler
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>PRX life</em>,  2024
    </div>
    <div class="periodical">
      <a href="https://doi.org/10.48550/arXiv.2402.09209" rel="external nofollow noopener" target="_blank">arXiv</a>
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://journals.aps.org/prxlife/abstract/10.1103/PRXLife.2.033008" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Cells regulate fates and complex body plans using spatiotemporal signaling cascades that al- ter gene expression. Enhancers, short DNA sequences (50-150 base pairs), help coordinate these cascades by attracting regulatory proteins to enhance the transcription of distal genes by bind- ing to promoters. In humans, there are hundreds of thousands of enhancers dispersed across the genome, which poses a challenging coordination task to prevent unintended gene activation. To mit- igate this problem, the genome contains additional DNA elements, insulators, that block enhancer- promoter interactions. However, there is an open problem with how the insulation works, especially as enhancer-insulator pairs may be separated by millions of base pairs. Based on recent empirical data from Hi-C experiments, this paper proposes a new mechanism that challenges the common paradigm that rests on specific insulator-insulator interactions. Instead, this paper introduces a stochastic looping model where enhancers bind weakly to surrounding chromatin. After calibrating the model to experimental data, we use simulations to study the broad distribution of hitting times between an enhancer and a promter when there are blocking insulators. In some cases, there is a large difference between average and most probable hitting times, making it difficult to assign a typical time scale, hinting at highly defocused regulation times. We also map our computational model onto a resetting problem that allows us to derive several analytical results. Besides offering new insights into enhancer-insulator interactions, our paper advances the understanding of gene regulatory networks and causal connections between genome folding and gene activation.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="hedstrom2024identifying" class="col-sm-8">
    <!-- Title -->
    <div class="title">Identifying stable communities in Hi-C using multifractal network modularity</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Lucas
            Hedström
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Antón Carcedo
            Martínez
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>arXiv:2405.05425</em>,  2024
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://arxiv.org/abs/2405.05425" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Chromosome capture techniques like Hi-C have expanded our understanding of mammalian genome 3D architecture and how it influences gene activity. To analyze Hi-C data sets, researchers increasingly treat them as DNA-contact networks and use standard community detection techniques to identify mesoscale 3D communities. However, there are considerable challenges in finding significant communities because the Hi-C networks have cross-scale interactions and are almost fully connected.  This paper presents a pipeline to distill 3D communities that remain intact under experimental noise. To this end, we bootstrap an ensemble of Hi-C datasets representing noisy data and extract 3D communities that we compare with the unperturbed dataset. Notably, we extract the communities by maximizing local modularity (using the Generalized Louvain method), which considers the multifractal spectrum recently discovered in Hi-C maps. Our pipeline finds that stable communities (under noise) typically have above-average internal contact frequencies and tend to be enriched in active chromatin marks. We also find they fold into more nested cross-scale hierarchies than less stable ones. Apart from presenting how to systematically extract robust communities in Hi-C data, our paper offers new ways to generate null models that take advantage of the network’s multifractal properties. We anticipate this has a broad applicability to several network applications.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="lee2024general" class="col-sm-8">
    <!-- Title -->
    <div class="title">General Resetting Theory for Group Avoidance</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Juhee
            Lee
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Seong-Gyu
            Yang
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Hye Jin
            Park
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 1 more author" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '1 more author' ? 'Ludvig Lizana' : '1 more author';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">1 more author</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>arXiv:2412.02524</em>,  2024
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://arxiv.org/abs/2412.02524" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>We present a general theoretical framework for group resetting dynamics in multi-agent systems in a drift potential. This setup contrasts with a typical resetting problem that involves a single searcher looking for a target, with resetting traditionally studied to optimize the search time to a target. More recently, resetting has also been used as a regulatory mechanism to avoid adverse outcomes, such as preventing critically high water levels in dams or deleveraging financial portfolios. Here, we extend current resetting theories to group dynamics, with applications ranging from bacterial evolution under antibiotic pressure to multiple-searcher optimization algorithms. Our framework incorporates extreme value statistics and renewal theory, from which we derive a master equation for the center of mass distribution of a group of searchers. This master equation allows us to calculate essential observables analytically. For example, how the group’s average position depends on group size, resetting rates, drift potential strength, and diffusion constants. This theoretical approach offers a new perspective on optimizing group search and regulatory mechanisms through resetting.</p>
      </div>
    

    

    
  </div>
</div>
</li>
</ol>
<h2 class="bibliography">2023</h2>
<ol class="bibliography">
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="lizana2023polycomb" class="col-sm-8">
    <!-- Title -->
    <div class="title">Polycomb proteins translate histone methylation to chromatin folding</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Negar
            Nahali
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Yuri B
            Schwartz
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Journal of Biological Chemistry</em>,  2023
    </div>
    <div class="periodical">
      <a href="https://www.biorxiv.org/content/10.1101/2022.05.13.491808v2" rel="external nofollow noopener" target="_blank">bioRxiv</a>
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://www.jbc.org/article/S0021-9258(23)02108-7/fulltext" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Epigenetic repression often involves covalent histone modifications. Yet, how the presence of a histone mark translates into changes in chromatin structure that ultimately benefits the repression is largely unclear. Polycomb group proteins comprise a family of evolutionarily conserved epigenetic repressors. They act as multi-subunit complexes one of which tri-methylates histone H3 at Lysine 27 (H3K27). Here we describe a novel Monte Carlo–Molecular Dynamics simulation framework, which we employed to discover that stochastic interaction of Polycomb Repressive Complex 1 (PRC1) with tri-methylated H3K27 is sufficient to fold the methylated chromatin. Unexpectedly, such chromatin folding leads to spatial clustering of the DNA elements bound by PRC1. Our results provide further insight into mechanisms of epigenetic repression and the process of chromatin folding in response to histone methylation.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="bernenko2023mapping" class="col-sm-8">
    <!-- Title -->
    <div class="title">Mapping the semi-nested community structure of 3D chromosome contact networks</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Dolores
            Bernenko
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Sang Hoon
            Lee
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Per
            Stenberg
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 1 more author" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '1 more author' ? 'Ludvig Lizana' : '1 more author';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">1 more author</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>PLOS Computational Biology</em>,  2023
    </div>
    <div class="periodical">
      <a href="https://www.biorxiv.org/content/10.1101/2022.06.24.497560v2" rel="external nofollow noopener" target="_blank">bioRxiv</a>
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011185" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Mammalian DNA folds into 3D structures that facilitate and regulate genetic processes such as transcription, DNA repair, and epigenetics. Several insights derive from chromosome capture methods, such as Hi-C, which allow researchers to construct contact maps depicting 3D interactions among all DNA segment pairs. These maps show a complex cross-scale organization spanning megabase-pair compartments to short-ranged DNA loops. To better understand the organizing principles, several groups analyzed Hi-C data assuming a Russian-doll-like nested hierarchy where DNA regions of similar sizes merge into larger and larger structures. Apart from being a simple and appealing description, this model explains, e.g., the omnipresent chequerboard pattern seen in Hi-C maps, known as A/B compartments, and foreshadows the co-localization of some functionally similar DNA regions. However, while successful, this model is incompatible with the two competing mechanisms that seem to shape a significant part of the chromosomes’ 3D organization: loop extrusion and phase separation. This paper aims to map out the chromosome’s actual folding hierarchy from empirical data. To this end, we take advantage of Hi-C experiments and treat the measured DNA-DNA interactions as a weighted network. From such a network, we extract 3D communities using the generalized Louvain algorithm. This algorithm has a resolution parameter that allows us to scan seamlessly through the community size spectrum, from A/B compartments to topologically associated domains (TADs). By constructing a hierarchical tree connecting these communities, we find that chromosomes are more complex than a perfect hierarchy. Analyzing how communities nest relative to a simple folding model, we found that chromosomes exhibit a significant portion of nested and non-nested community pairs alongside considerable randomness. In addition, by examining nesting and chromatin types, we discovered that nested parts are often associated with active chromatin. These results highlight that cross-scale relationships will be essential components in models aiming to reach a deep understanding of the causal mechanisms of chromosome folding.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="hedstrom2023modelling" class="col-sm-8">
    <!-- Title -->
    <div class="title">Modelling chromosome-wide target search</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Lucas
            Hedström
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>New Journal of Physics</em>,  2023
    </div>
    <div class="periodical">
      <a href="https://arxiv.org/abs/2211.04195" rel="external nofollow noopener" target="_blank">arXiv</a>
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://iopscience.iop.org/article/10.1088/1367-2630/acc127/meta" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>The most common gene regulation mechanism is when a transcription factor (TF) protein binds to a regulatory sequence to increase or decrease RNA transcription. However, TFs face two main challenges when searching for these sequences. First, the sequences are vanishingly short relative to the genome length. Second, there are many nearly identical sequences scattered across the genome, causing proteins to suspend the search. But as pointed out in a computational study of LacI regulation in Escherichia coli, such almost-targets may lower search times if considering DNA looping. In this paper, we explore if this also occurs over chromosome-wide distances. To this end, we developed a cross-scale computational framework that combines established facilitated-diffusion models for basepair-level search and a network model capturing chromosome-wide leaps. To make our model realistic, we used Hi-C data sets as a proxy for 3D proximity between long-ranged DNA segments and binding profiles for more than 100 TFs. Using our cross-scale model, we found that median search times to individual targets critically depend on a network metric combining node strength (sum of link weights) and local dissociation rates. Also, by randomizing these rates, we found that some actual 3D target configurations stand out as considerably faster or slower than their random counterparts. This finding hints that chromosomes’ 3D structure funnels essential TFs to relevant DNA regions.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="holmgren2023mapping" class="col-sm-8">
    <!-- Title -->
    <div class="title">Mapping robust multiscale communities in chromosome contact networks</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Anton
            Holmgren
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Dolores
            Bernenko
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Scientific Reports</em>,  2023
    </div>
    <div class="periodical">
      <a href="hhttps://arxiv.org/abs/2212.08456">arXiv</a>
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://www.nature.com/articles/s41598-023-39522-7" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>To better understand DNA’s 3D folding in cell nuclei, researchers developed chromosome capture methods such as Hi-C that measure the contact frequencies between all DNA segment pairs across the genome. As Hi-C data sets often are massive, it is common to use bioinformatics methods to group DNA segments into 3D regions with correlated contact patterns, such as Topologically associated domains and A/B compartments. Recently, another research direction emerged that treats the Hi-C data as a network of 3D contacts. In this representation, one can use community detection algorithms from complex network theory that group nodes into tightly connected mesoscale communities. However, because Hi-C networks are so densely connected, several node partitions may represent feasible solutions to the community detection problem but are indistinguishable unless including other data. Because this limitation is a fundamental property of the network, this problem persists regardless of the community-finding or data-clustering method. To help remedy this problem, we developed a method that charts the solution landscape of network partitions in Hi-C data from human cells. Our approach allows us to scan seamlessly through the scales of the network and determine regimes where we can expect reliable community structures. We find that some scales are more robust than others and that strong clusters may differ significantly. Our work highlights that finding a robust community structure hinges on thoughtful algorithm design or method cross-evaluation.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="bernenko2023exploring" class="col-sm-8">
    <!-- Title -->
    <div class="title">Exploring 3D community inconsistency in human chromosome contact networks</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Dolores
            Bernenko
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Sang Hoon
            Lee
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Journal of Physics: Complexity</em>,  2023
    </div>
    <div class="periodical">
      <a href="hhttps://arxiv.org/abs/2302.14684">arXiv</a>
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://iopscience.iop.org/article/10.1088/2632-072X/acef9d" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Researchers have developed chromosome capture methods such as Hi-C to better understand DNA’s 3D folding in nuclei. The Hi-C method captures contact frequencies between DNA segment pairs across the genome. When analyzing Hi-C data sets, it is common to group these pairs using standard bioinformatics methods (e.g. PCA). Other approaches handle Hi-C data as weighted networks, where connected node pairs represent DNA segments in 3D proximity. In this representation, one can leverage community detection techniques developed in complex network theory to group nodes into mesoscale communities containing nodes with similar connection patterns. While there are several successful attempts to analyze Hi-C data in this way, it is common to report and study the most typical community structure. But in reality, there are often several valid candidates. Therefore, depending on algorithm design, different community detection methods focusing on slightly different connectivity features may have differing views on the ideal node groupings. In fact, even the same community detection method may yield different results if using a stochastic algorithm. This ambiguity is fundamental to community detection and shared by most complex networks whenever interactions span all scales in the network. This is known as community inconsistency. This paper explores this inconsistency of 3D communities in Hi-C data for all human chromosomes. We base our analysis on two inconsistency metrics, one local and one global, and quantify the network scales where the community separation is most variable. For example, we find that TADs are less reliable than A/B compartments and that nodes with highly variable node-community memberships are associated with open chromatin. Overall, our study provides a helpful framework for data-driven researchers and increases awareness of some inherent challenges when clustering Hi-C data into 3D communities.</p>
      </div>
    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="lundkvist2023forecasting" class="col-sm-8">
    <!-- Title -->
    <div class="title">Forecasting histone methylation by Polycomb complexes with minute-scale precision</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Moa J.
            Lundkvist
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Yuri B.
            Schwartz
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Science Advances</em>,  2023
    </div>
    <div class="periodical">
      <a href="https://www.biorxiv.org/content/10.1101/2023.06.16.545270v1.abstract" rel="external nofollow noopener" target="_blank">bioRxiv</a>
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://www.science.org/doi/abs/10.1126/sciadv.adj8198" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Animals use the Polycomb system to epigenetically repress developmental genes. The repression requires trimethylation of lysine 27 of histone H3 (H3K27me3) by Polycomb Repressive Complex 2 (PRC2), but the dynamics of this process is poorly understood. To bridge the gap, we developed a computational model that forecasts H3K27 methylation in Drosophila with high temporal resolution and spatial accuracy of contemporary experimental techniques. Using this model, we show that pools of methylated H3K27 in dividing cells are defined by the effective concentration of PRC2 and the replication frequency. We find that the allosteric stimulation by preexisting H3K27me3 makes PRC2 better in methylating developmental genes as opposed to indiscriminate methylation throughout the genome. Applied to Drosophila development, our model argues that, in this organism, the intergenerationally inherited H3K27me3 does not “survive” rapid cycles of embryonic chromatin replication and is unlikely to transmit the memory of epigenetic repression to the offspring. Our model is adaptable to other organisms, including mice and humans. Computational forecasts of histone methylation shed light on the intergenerational inheritance of epigenetic repression.</p>
      </div>
    

    

    
  </div>
</div>
</li>
</ol>
<h2 class="bibliography">2022</h2>
<ol class="bibliography"><li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="kim2022fractal" class="col-sm-8">
    <!-- Title -->
    <div class="title">Fractal and knot-free chromosomes facilitate nucleoplasmic transport</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Yeonghoon
            Kim
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Jae-Hyung
            Jeon
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Physical Review Letters</em>,  2022
    </div>
    <div class="periodical">
      <a href="https://arxiv.org/abs/2109.14933" rel="external nofollow noopener" target="_blank">arXiv</a>
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.128.038101" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Chromosomes in the nucleus assemble into hierarchies of 3D domains that, during interphase, share essential features with a knot-free condensed polymer known as the fractal globule (FG). The FG-like chromosome likely affects macromolecular transport, yet its characteristics remain poorly understood. Using computer simulations and scaling analysis, we show that the 3D folding and macromolecular size of the chromosomes determine their transport characteristics. Large-scale subdiffusion occurs at a critical particle size where the network of accessible volumes is critically connected. Condensed chromosomes have connectivity networks akin to simple Bernoulli bond percolation clusters, regardless of the polymer models. However, even if the network structures are similar, the tracer’s walk dimension varies. It turns out that the walk dimension depends on the network topology of the accessible volume and dynamic heterogeneity of the tracer’s hopping rate. We find that the FG structure has a smaller walk dimension than other random geometries, suggesting that the FG-like chromosome structure accelerates macromolecular diffusion and target-search.</p>
      </div>
    

    

    
  </div>
</div>
</li></ol>
<h2 class="bibliography">2021</h2>
<ol class="bibliography"><li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="nyberg2021modeling" class="col-sm-8">
    <!-- Title -->
    <div class="title">Modeling protein target search in human chromosomes</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Markus
            Nyberg
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Tobias
            Ambjörnsson
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Per
            Stenberg
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 1 more author" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '1 more author' ? 'Ludvig Lizana' : '1 more author';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">1 more author</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Physical Review Research</em>,  2021
    </div>
    <div class="periodical">
      <a href="https://arxiv.org/abs/1909.09478" rel="external nofollow noopener" target="_blank">arXiv</a>
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
        <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
      
      
      
      
        <a href="https://journals.aps.org/prresearch/abstract/10.1103/PhysRevResearch.3.013055" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a>
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    
      <!-- Hidden abstract block -->
      <div class="abstract hidden">
        <p>Several processes in the cell, such as gene regulation, start when key proteins recognize and bind to short DNA sequences. However, as these sequences can be hundreds of million times shorter than the genome, they are hard to find by simple diffusion: diffusion-limited association rates may underestimate in vitro measurements up to several orders of magnitude. Moreover, the rates increase if the DNA is coiled rather than straight. Here we model how this works in vivo in mammalian cells. We use chromatin-chromatin contact data from Hi-C experiments to map the protein target-search onto a network problem. The nodes represent DNA segments and the weight of the links are proportional to measured contact probabilities. We then put forward a diffusion-reaction equation for the density of searching protein that allows us to calculate the association rates across the genome analytically. For segments where the rates are high, we find that they are enriched with active gene starts and have high RNA expression levels. This paper suggests that the DNA’s 3D conformation is important for protein search times in vivo and offers a method to interpret protein-binding profiles in eukaryotes that cannot be explained by the DNA sequence itself.</p>
      </div>
    

    

    
  </div>
</div>
</li></ol>
<h2 class="bibliography">2019</h2>
<ol class="bibliography">
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="lee2019mapping" class="col-sm-8">
    <!-- Title -->
    <div class="title">Mapping the spectrum of 3D communities in human chromosome conformation capture data</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Sang Hoon
            Lee
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Yeonghoon
            Kim
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Sungmin
            Lee
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 4 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '4 more authors' ? 'Xavier Durang, Per Stenberg, Jae-Hyung Jeon, Ludvig Lizana' : '4 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">4 more authors</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Scientific reports</em>,  2019
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="kumar2019genomic" class="col-sm-8">
    <!-- Title -->
    <div class="title">Genomic 3D compartments emerge from unfolding mitotic chromosomes</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Rajendra
            Kumar
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Per
            Stenberg
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Chromosoma</em>,  2019
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="sobhy2019highly" class="col-sm-8">
    <!-- Title -->
    <div class="title">Highly interacting regions of the human genome are enriched with enhancers and bound by DNA repair proteins</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Haitham
            Sobhy
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Rajendra
            Kumar
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Jacob
            Lewerentz
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 2 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '2 more authors' ? 'Ludvig Lizana, Per Stenberg' : '2 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">2 more authors</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Scientific reports</em>,  2019
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="durang2019first" class="col-sm-8">
    <!-- Title -->
    <div class="title">First-passage statistics under stochastic resetting in bounded domains</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Xavier
            Durang
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Sungmin
            Lee
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 1 more author" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '1 more author' ? 'Jae-Hyung Jeon' : '1 more author';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">1 more author</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Journal of Physics A: Mathematical and Theoretical</em>,  2019
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
</ol>
<h2 class="bibliography">2018</h2>
<ol class="bibliography">
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="nyberg2018zero" class="col-sm-8">
    <!-- Title -->
    <div class="title">Zero-crossing statistics for non-Markovian time series</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Markus
            Nyberg
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Tobias
            Ambjörnsson
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Physical Review E</em>,  2018
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="nyberg2018persistence" class="col-sm-8">
    <!-- Title -->
    <div class="title">Persistence of non-Markovian Gaussian stationary processes in discrete time</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Markus
            Nyberg
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Physical Review E</em>,  2018
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
</ol>
<h2 class="bibliography">2017</h2>
<ol class="bibliography">
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="vasilyev2017smoluchowski" class="col-sm-8">
    <!-- Title -->
    <div class="title">Smoluchowski rate for diffusion-controlled reactions of molecules with antenna</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Oleg A
            Vasilyev
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Gleb
            Oshanin
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Journal of Physics A: Mathematical and Theoretical</em>,  2017
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="kumar2017genome" class="col-sm-8">
    <!-- Title -->
    <div class="title">Genome contact map explorer: a platform for the comparison, interactive visualization and analysis of genome contact maps</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Rajendra
            Kumar
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Haitham
            Sobhy
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Per
            Stenberg
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 1 more author" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '1 more author' ? 'Ludvig Lizana' : '1 more author';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">1 more author</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Nucleic acids research</em>,  2017
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
</ol>
<h2 class="bibliography">2016</h2>
<ol class="bibliography">
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="nyberg2016simple" class="col-sm-8">
    <!-- Title -->
    <div class="title">A simple method to calculate first-passage time densities with arbitrary initial conditions</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Markus
            Nyberg
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Tobias
            Ambjörnsson
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>New Journal of Physics</em>,  2016
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="lizana2016tracer" class="col-sm-8">
    <!-- Title -->
    <div class="title">Tracer particles in two-dimensional elastic networks diffuse logarithmically slow</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Tobias
            Ambjörnsson
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Michael A
            Lomholt
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Journal of Physics A: Mathematical and Theoretical</em>,  2016
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
</ol>
<h2 class="bibliography">2015</h2>
<ol class="bibliography"><li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="ahlberg2015many" class="col-sm-8">
    <!-- Title -->
    <div class="title">Many-body effects on tracer particle diffusion with applications for single-protein dynamics on DNA</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Sebastian
            Ahlberg
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Tobias
            Ambjörnsson
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>New Journal of Physics</em>,  2015
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li></ol>
<h2 class="bibliography">2014</h2>
<ol class="bibliography">
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="lizana2014single" class="col-sm-8">
    <!-- Title -->
    <div class="title">Single-file diffusion with non-thermal initial conditions</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Michael A
            Lomholt
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Tobias
            Ambjörnsson
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Physica A: Statistical Mechanics and its Applications</em>,  2014
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="lizana2014exact" class="col-sm-8">
    <!-- Title -->
    <div class="title">Exact expressions for the mobility and electrophoretic mobility of a weakly charged sphere in a simple electrolyte</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Alexander Y
            Grosberg
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Europhysics Letters</em>,  2014
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="mirshahvalad2014dynamics" class="col-sm-8">
    <!-- Title -->
    <div class="title">Dynamics of interacting information waves in networks</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Atieh
            Mirshahvalad
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Alcides Viamontes
            Esquivel
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 1 more author" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '1 more author' ? 'Martin Rosvall' : '1 more author';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">1 more author</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Physical Review E</em>,  2014
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="forsling2014non" class="col-sm-8">
    <!-- Title -->
    <div class="title">Non-Markovian effects in the first-passage dynamics of obstructed tracer particle diffusion in one-dimensional systems</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Robin
            Forsling
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Lloyd P
            Sanders
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Tobias
            Ambjörnsson
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 1 more author" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '1 more author' ? 'Ludvig Lizana' : '1 more author';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">1 more author</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>The Journal of Chemical Physics</em>,  2014
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="sanders2014severe" class="col-sm-8">
    <!-- Title -->
    <div class="title">Severe slowing-down and universality of the dynamics in disordered interacting many-body systems: ageing and ultraslow diffusion</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Lloyd P
            Sanders
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Michael A
            Lomholt
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 3 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '3 more authors' ? 'Karl Fogelmark, Ralf Metzler, Tobias Ambjörnsson' : '3 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">3 more authors</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>New Journal of Physics</em>,  2014
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="metzler2014ageing" class="col-sm-8">
    <!-- Title -->
    <div class="title">Ageing single file motion</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Ralf
            Metzler
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Lloyd
            Sanders
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Michael Andersen
            Lomholt
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 3 more authors" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '3 more authors' ? 'Ludvig Lizana, Karl Fogelmark, Tobias Ambjörnsson' : '3 more authors';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">3 more authors</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>The European Physical Journal Special Topics</em>,  2014
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="forsling2014first" class="col-sm-8">
    <!-- Title -->
    <div class="title">First-passage dynamics of obstructed tracer particle diffusion in one-dimensional systems</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Robin
            Forsling
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Lloyd
            Sanders
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Tobias
            Ambjörnsson
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 1 more author" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '1 more author' ? 'Ludvig Lizana' : '1 more author';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">1 more author</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>arXiv preprint arXiv:1401.2933</em>,  2014
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
</ol>
<h2 class="bibliography">2013</h2>
<ol class="bibliography">
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="pigolotti2013quality" class="col-sm-8">
    <!-- Title -->
    <div class="title">Quality control system response to stochastic growth of amyloid fibrils</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Simone
            Pigolotti
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Daniel
            Otzen
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 1 more author" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '1 more author' ? 'Kim Sneppen' : '1 more author';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">1 more author</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>FEBS letters</em>,  2013
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="lund2013effects" class="col-sm-8">
    <!-- Title -->
    <div class="title">Effects of city-size heterogeneity on epidemic spreading in a metapopulation: a reaction-diffusion approach</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Halvor
            Lund
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Ingve
            Simonsen
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Journal of statistical physics</em>,  2013
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="lomholt2013microscopic" class="col-sm-8">
    <!-- Title -->
    <div class="title">Microscopic origin of the logarithmic time evolution of aging processes in complex systems</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Michael A
            Lomholt
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ralf
            Metzler
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 1 more author" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '1 more author' ? 'Tobias Ambjörnsson' : '1 more author';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">1 more author</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Physical review letters</em>,  2013
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
</ol>
<h2 class="bibliography">2011</h2>
<ol class="bibliography">
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="lomholt2011dissimilar" class="col-sm-8">
    <!-- Title -->
    <div class="title">Dissimilar bouncy walkers</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Michael A
            Lomholt
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Tobias
            Ambjörnsson
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>The Journal of chemical physics</em>,  2011
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="lizana2011modeling" class="col-sm-8">
    <!-- Title -->
    <div class="title">Modeling the spatial dynamics of culture spreading in the presence of cultural strongholds</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Ludvig
            Lizana
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Namiko
            Mitarai
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Kim
            Sneppen
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 1 more author" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '1 more author' ? 'Hiizu Nakanishi' : '1 more author';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">1 more author</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>Physical Review E</em>,  2011
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
<li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="bernhardsson2011structural" class="col-sm-8">
    <!-- Title -->
    <div class="title">Structural correlations in bacterial metabolic networks</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            Sebastian
            Bernhardsson
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            Philip
            Gerlee
          
        
      
        
        
        
        
        
        

        
          , 
          and 
        
        
          
            Ludvig
            Lizana
          
        
      
      

      
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>BMC evolutionary biology</em>,  2011
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li>
</ol>
<h2 class="bibliography">2010</h2>
<ol class="bibliography"><li>
<div class="row">
  
    <div class="col-sm-2 abbr">
      
    </div>
  

  <!-- Entry bib key -->
  <div id="gerlee2010structural" class="col-sm-8">
    <!-- Title -->
    <div class="title">Structural correlations in bacterial metabolic networks</div>
    <!-- Author -->
    <div class="author">
      

      
      

      
        
        
        
        
        
        

        
          
          
        
        
          
            P
            Gerlee
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            S
            Bernhardsson
          
        
      
        
        
        
        
        
        

        
          , 
          
        
        
          
            L
            Lizana
          
        
      
      

      
      
        
        
        
        , and
        <span class="more-authors" title="click to view 1 more author" onclick="
              var element = $(this);
              element.attr('title', '');
              var more_authors_text = element.text() == '1 more author' ? 'K Sneppen' : '1 more author';
              var cursorPosition = 0;
              var textAdder = setInterval(function(){
                element.text(more_authors_text.substring(0, cursorPosition + 1));
                if (++cursorPosition == more_authors_text.length){
                  clearInterval(textAdder);
                }
            }, '10');
          ">1 more author</span>
      
    </div>

    <!-- Journal/Book title and date -->
    
    
    
    
    
    
    
    <div class="periodical">
      <em>In FEBS JOURNAL</em> ,  2010
    </div>
    <div class="periodical">
      
    </div>

    <!-- Links/Buttons -->
    <div class="links">
      
      
      
      
      
      
      
      
      
      
      
      
    </div>
    
      
      
      
    

    

    

    
  </div>
</div>
</li></ol>

</div>

  </article>

  

  
</div>

      
    </div>

    <!-- Footer -->
    
  <footer class="fixed-bottom" role="contentinfo">
    <div class="container mt-0">
      © Copyright 2025
      Ludvig
      
      Lizana. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a>. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>.

      
      
    </div>
  </footer>



    <!-- JavaScripts -->
    <!-- jQuery -->
<script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
<script src="/assets/js/bootstrap.bundle.min.js"></script>
<!-- <script src="/assets/js/mdb.min.js"></script> -->
<script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    
  <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>


    

    

    

    

    

    

  <!-- Medium Zoom JS -->
  <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script>
  <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script>



<!-- Bootstrap Table -->
<script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script>

<!-- Load Common JS -->
<script src="/assets/js/no_defer.js?2930004b8d7fcd0a8e00fdcfc8fc9f24"></script>
<script defer src="/assets/js/common.js?4a129fbf39254905f505c7246e641eaf"></script>
<script defer src="/assets/js/copy_code.js?906085072389244b4db0c89c5b1dd28d" type="text/javascript"></script>

    
  <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>


  <script async src="https://badge.dimensions.ai/badge.js"></script>


    
  <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams',
      },
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>


    


    
  <!-- Scrolling Progress Bar -->
  <script type="text/javascript">
    /*
     * This JavaScript code has been adapted from the article
     * https://css-tricks.com/reading-position-indicator/ authored by Pankaj Parashar,
     * published on the website https://css-tricks.com on the 7th of May, 2014.
     * Couple of changes were made to the original code to make it compatible
     * with the `al-foio` theme.
     */
    const progressBar = $('#progress');
    /*
     * We set up the bar after all elements are done loading.
     * In some cases, if the images in the page are larger than the intended
     * size they'll have on the page, they'll be resized via CSS to accomodate
     * the desired size. This mistake, however, breaks the computations as the
     * scroll size is computed as soon as the elements finish loading.
     * To account for this, a minimal delay was introduced before computing the
     * values.
     */
    window.onload = function () {
      setTimeout(progressBarSetup, 50);
    };
    /*
     * We set up the bar according to the browser.
     * If the browser supports the progress element we use that.
     * Otherwise, we resize the bar thru CSS styling
     */
    function progressBarSetup() {
      if ('max' in document.createElement('progress')) {
        initializeProgressElement();
        $(document).on('scroll', function () {
          progressBar.attr({ value: getCurrentScrollPosition() });
        });
        $(window).on('resize', initializeProgressElement);
      } else {
        resizeProgressBar();
        $(document).on('scroll', resizeProgressBar);
        $(window).on('resize', resizeProgressBar);
      }
    }
    /*
     * The vertical scroll position is the same as the number of pixels that
     * are hidden from view above the scrollable area. Thus, a value > 0 is
     * how much the user has scrolled from the top
     */
    function getCurrentScrollPosition() {
      return $(window).scrollTop();
    }

    function initializeProgressElement() {
      let navbarHeight = $('#navbar').outerHeight(true);
      $('body').css({ 'padding-top': navbarHeight });
      $('progress-container').css({ 'padding-top': navbarHeight });
      progressBar.css({ top: navbarHeight });
      progressBar.attr({
        max: getDistanceToScroll(),
        value: getCurrentScrollPosition(),
      });
    }
    /*
     * The offset between the html document height and the browser viewport
     * height will be greater than zero if vertical scroll is possible.
     * This is the distance the user can scroll
     */
    function getDistanceToScroll() {
      return $(document).height() - $(window).height();
    }

    function resizeProgressBar() {
      progressBar.css({ width: getWidthPercentage() + '%' });
    }
    // The scroll ratio equals the percentage to resize the bar
    function getWidthPercentage() {
      return (getCurrentScrollPosition() / getDistanceToScroll()) * 100;
    }
  </script>


    

  </body>
</html>
